bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/handler/ActionHandler.java

```diff
@@ -17,11 +17,10 @@
 import org.eclipse.jdt.annotation.NonNullByDefault;
 import org.eclipse.jdt.annotation.Nullable;
 import org.eclipse.smarthome.automation.Action;
-import org.eclipse.smarthome.automation.Rule;
 
 /**
  * This interface is implemented by external modules which are called by the
- * RuleEngine to execute {@link Action}s of the {@link Rule}s.
+ * RuleManager to execute {@link Action}s of the {@link RuleImpl}s.
```

Comment:
If I use a custom implementation of the `Rule` interface I assume that this manager still work for other implementations.
Rule instead of RuleImpl?